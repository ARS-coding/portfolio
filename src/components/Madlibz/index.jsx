import React from 'react';
import { Col, Row } from 'react-bootstrap';
import { VerticalTimelineElement } from 'react-vertical-timeline-component';

const Juniper = () => (
  <VerticalTimelineElement
    className="vertical-timeline-element--work"
    date="Aug 2020"
    dateClassName="date"
    contentStyle={{
      background: 'white',
      color: 'white',
    }}
    contentArrowStyle={{
      borderRight: '7px solid  white',
    }}
    iconStyle={{
      background: 'white',
    }}
    icon={<i className="fas fa-project-diagram projectIcon"></i>}
  >
    <div className=" projectContent">
      <h4>Madlibz</h4>
      <small>Team Project</small> <br />
      <a
        href="https://github.com/tmansuroglu/Madlibzz"
        target="_blank"
        rel="noreferrer"
      >
        Github
      </a>{' '}
      |{' '}
      <a href="https://madlibz.netlify.app/" target="_blank" rel="noreferrer">
        Demo
      </a>
      <p className="projectText">
        <strong>Goal:</strong> creating Mad Libs game as an introduction to
        project building.
      </p>
      <br />
      <Row>
        <Col xs={12} md={6}>
          <h5>Skills Applied</h5>
          <ul>
            <li>HTML</li>
            <li>CSS</li>
            <li>JavaScript</li>
            <li>Regular Expression</li>
            <li>i18n</li>
            <li>Bootstrap</li>
            <li>Visual Design</li>
            <li>Responsive Design</li>
            <li>Teamwork</li>
          </ul>
        </Col>
        <Col xs={12} md={6}>
          <h5>My Responsibilities</h5>
          <ul>
            <li>Game Logic</li>
            <li>Setting up i18n</li>
            <li>Visual Design</li>
            <li>Responsive Design</li>
          </ul>
        </Col>
      </Row>
    </div>
  </VerticalTimelineElement>
);

export default Juniper;
