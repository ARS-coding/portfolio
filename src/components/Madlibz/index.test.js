import React from 'react';
import renderer from 'react-test-renderer';
import Madlibz from '.';

it('Madlibz renders correctly', () => {
  const tree = renderer.create(<Madlibz />).toJSON();
  expect(tree).toMatchSnapshot();
});
