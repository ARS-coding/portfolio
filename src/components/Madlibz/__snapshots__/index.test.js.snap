// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Madlibz renders correctly 1`] = `
<div
  className="vertical-timeline-element--work vertical-timeline-element"
  id=""
  style={null}
>
  <span
    className="vertical-timeline-element-icon is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
      }
    }
  >
    <i
      className="fas fa-project-diagram projectIcon"
    />
  </span>
  <div
    className="vertical-timeline-element-content is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
        "color": "white",
      }
    }
  >
    <div
      className="vertical-timeline-element-content-arrow"
      style={
        Object {
          "borderRight": "7px solid  white",
        }
      }
    />
    <div
      className="homeText projectContent"
    >
      <h4>
        Madlibz
      </h4>
      <a
        href="https://github.com/tmansuroglu/Madlibzz"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
       
      |
       
      <a
        href="https://madlibz.netlify.app/"
        rel="noreferrer"
        target="_blank"
      >
        Demo
      </a>
      <p>
        <strong>
          Goal:
        </strong>
         creating a Madlibz game.
      </p>
      <br />
      <div
        className="row"
      >
        <div
          className="col-md-6 col-12"
        >
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              Regular Expression
            </li>
            <li>
              i18n
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Visual Design
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Teamwork
            </li>
          </ul>
        </div>
        <div
          className="col-md-6 col-12"
        >
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Game Logic
            </li>
            <li>
              Setting up i18n
            </li>
            <li>
              Visual Design
            </li>
            <li>
              Responsive Design
            </li>
          </ul>
        </div>
      </div>
    </div>
    <span
      className="date vertical-timeline-element-date"
    >
      Aug 2020
    </span>
  </div>
</div>
`;
